### PR TITLE
Remove preinstalled apps from Changes and Releases

### PR DESCRIPTION
**Pull request by marians**
Towards https://github.com/giantswarm/giantswarm/issues/14712

### CODE REVIEW

File: scripts/aggregate-changelogs/config.yaml [hunk 1]
```diff
@@ -20,30 +20,20 @@ repositories:
...
     category: Control Plane Kubernetes API
-  giantswarm/cert-exporter:
-    category: Managed Apps
   giantswarm/cert-manager-app:
     category: Managed Apps
-  giantswarm/chart-operator:
-    category: Managed Apps
-  giantswarm/cluster-autoscaler-app:
-    category: Managed Apps
   giantswarm/coredns-app:
```
Comment (cokiengchiara) [inline scripts/aggregate-changelogs/config.yaml:25]: this is a preinstalled app too

File: scripts/aggregate-changelogs/config.yaml [hunk 2]
```diff
@@ -52,8 +42,6 @@ repositories:
     category: Managed Apps
   giantswarm/net-exporter:
```
Comment (cokiengchiara) [inline scripts/aggregate-changelogs/config.yaml:43]: this is a preinstalled app too

**Review by cokiengchiara — CHANGES_REQUESTED** [review]
Thank you! Please remove coredns and net exporter too :)

**Comment by marians** [comment]
@cokiengchiara Done

**Review by cokiengchiara — APPROVED** [review]
thanks so much! LGTM

**Pull request merged**